packages/@react-spectrum/dialog/src/DialogTrigger.tsx

```diff
@@ -149,7 +143,7 @@ function PopoverTrigger({isOpen, onPress, onClose, targetRef, trigger, content,
 
   let overlay = (
     <Overlay isOpen={isOpen}>
-      <Popover {...popoverProps} ref={overlayRef} onClose={onClose} placement={placement} arrowProps={arrowProps} hideArrow={hideArrow}>
+      <Popover UNSAFE_style={popoverProps.style} ref={overlayRef} onClose={onClose} placement={placement} arrowProps={arrowProps} hideArrow={hideArrow}>
```

Comment:
shouldn't we change popoverProps to use `UNSAFE_style` instead of `style`? And pass things in via `useStyleProps`?

Reply:
well, I want to keep aria returning real DOM props. So, translating here.